src/include/duckdb/parser/parsed_data/copy_database_info.hpp

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/parser/parsed_data/copy_database_info.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/common/unique_ptr.hpp"
+#include "duckdb/common/vector.hpp"
+#include "duckdb/parser/parsed_data/create_info.hpp"
+#include "duckdb/parser/parsed_data/parse_info.hpp"
+
+namespace duckdb {
+
+struct CopyDatabaseInfo : public ParseInfo {
+public:
+	static constexpr const ParseInfoType TYPE = ParseInfoType::COPY_DATABASE_INFO;
+
+public:
+	explicit CopyDatabaseInfo() : ParseInfo(TYPE), target_database(INVALID_CATALOG) {
+	}
+
+	explicit CopyDatabaseInfo(const std::string &target_database) : ParseInfo(TYPE), target_database(target_database) {
+	}
+
+	// The destination database to which catalog entries are being copied
+	std::string target_database;
```

Comment:
nit: please avoid the `std::` prefix and use our own string type

Reply:
I've removed the prefix.